### PR TITLE
Region selection

### PR DESCRIPTION
**Pull request by lattemilk**
# Changes:
* Tweaked the top bar to match the mocks more closely
* Added region selection component
* Added types directory
* Added region enum
* Changed default path to region selection

# Expectations:
- [ ] Hovering over Sierra Nevada should change the icon color
- [ ] Other regions should be disabled
- [ ] Clicking Sierra Nevada sets the region in the component

# UI Update: 
Screen Shot 2022-10-27 at 7 02 52 PM

# Future Changes:
* Pass region data
* Open up explore component to selected region
* Add region selection dropdown to top bar

### CODE REVIEW

File: src/interface/src/app/region-selection/region-selection.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="center-block">
+  <section class="image-section">
+    <img src="assets/svg/planscape-image.svg">
+  </section>
+  <section class="content-section">
+    Different regions have varied datasets available.
```
Comment (riecke) [inline src/interface/src/app/region-selection/region-selection.component.html:6]: Suggested rewording: "Different regions are associated with different datasets from the Regional Resource Kits."

**Review by riecke — APPROVED** [review]

File: src/interface/src/app/region-selection/region-selection.component.html (hunk 1 above)
Comment (lattemilk) [reply src/interface/src/app/region-selection/region-selection.component.html:6]: Thanks for reviewing! I pulled the text from the mocks- I'll double check with UX?

Comment (lattemilk) [reply src/interface/src/app/region-selection/region-selection.component.html]: Checked with Alex and the wording will probably change once we get a UX Writer/Technical writer. Updated the text to your suggestion.

**Review by thequeenofspades — APPROVED** [review]
Looks great!

**Pull request merged**